src/applications/disability-benefits/all-claims/Form526EZApp.jsx

```diff
@@ -64,6 +65,13 @@ export const hasRequiredId = profile =>
 
 export const hasRequiredDob = profile => !!profile.dob;
 
+// Will replace hasRequiredServices, hasRequiredId and hasRequiredDob when form526RequiredIdentifiersInUserObject feature flag is removed
+const missingRequiredIdentifiers = profile => {
+  return Object.values(profile.claims.form526RequiredIdentifierPresence).some(
+    idPresence => idPresence === false,
```

Comment:
will this value always be `false`?  or is there a chance it will be `null`?  what would that do to this code?